src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public abstract class Machine {
+    public void doWork(){
+
+    }
+    public void stopWork() {
+
+    }
```

Comment:
The `stopWork()` method currently has no functionality. Since this is an abstract class, this method should be declared as abstract. This will enforce that any class extending `Machine` will have to provide an implementation for this method.